OrbitCore/OrbitModule.cpp

```diff
@@ -158,12 +158,27 @@ std::unique_ptr<ElfFile> FindSymbols(const std::string& module_path) {
 bool Pdb::LoadFunctions(const char* file_name) {
   m_LoadedModuleName = file_name;
   std::unique_ptr<ElfFile> elf_file = FindSymbols(file_name);
+
+  if (elf_file == nullptr) {
+    PRINT(absl::StrFormat("Unable to load elf-file \"%s\"", m_FileName));
```

Comment:
Thanks for catching that! It was unsafe in my code. 
Minor adjustment, because m_FileName might not be initialized 
```suggestion
    PRINT(absl::StrFormat("Unable to load elf-file \"%s\"", file_name));
```

Reply:
done.